newIDE/app/src/GameDashboard/GameAnalyticsPanel.js

```diff
@@ -142,6 +144,19 @@ export const GameAnalyticsPanel = ({
                 <Column noMargin alignItems="center" expand>
                   <Text size="block-title" align="center">
                     <Trans>{chartData.overview.playersCount} sessions</Trans>
+                    <Tooltip
+                      title={
+                        <Text>
+                          <Trans>
+                            Number of people who launched the game. A viewer is
+                            a single visitor, and a player is a visitor after 60
+                            seconds of activity in the game.
```

Comment:
```suggestion
                            Visitors are considered players when they stayed at least 60 seconds including loading screens.
```

Reply:
Ok for this change, just sticking with the word **Viewers** as the graph is using already this term.